src/main/java/com/scottlogic/competition_engine/application/TestController.java

```diff
@@ -0,0 +1,18 @@
+package com.scottlogic.competition_engine.application;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class TestController {
+
+    @Autowired
+    public TestController() {}
+
```

Comment:
I  dont think this PR should include this, the user endpoint and its tests will be in Geidre's PR